connector/docs/user_guide/services/vmanage.rst

```diff
@@ -0,0 +1,211 @@
+vManage
+=======
+
+The following services are supported by the REST connector for vManage.
+
+
+connect
+-------
+
+API to connect to the device.
+
+The vManage REST implementation supports specifying the port to connect to 
+and allows to specify the username and password in the testbed YAML file.
+
+.. code-block:: yaml
+
+    # Example
+    # -------
+    #
+    #   pyATS testbed yaml example for defining a REST connection
+
+    testbed:
+        name: myTestbed
+
+    devices:
+        vmanage:
+            os: sdwan
+            series: viptela
+            type: vmanage
+            custom:
+                abstraction:
+                    order: [type]
```

Comment:
I assume vManage is only controller for viptela devices. so maybe we can simply have like below just as `os: viptela` 
```suggestion
        vmanage:
            os: viptela
            custom:
                abstraction:
                    order: [os]
```
if we have something else which could have different implementation, another suggestion would be below and do abstraction with 2 keys (os and series).
```
```suggestion
        vmanage:
            os: viptela
            series: vmanage
            custom:
                abstraction:
                    order: [os, series]
```
what do you think?

Reply:
Thanks @tahigash, as existing unicon.plugin for viptela is using below structure, so may I have to use like this.
        vmanage:
            os: sdwan
            series: viptela
            type: vmanage

Reply:
@jeaubin @tahigash,  Do I have to change this part?
 I just want to keep os/series/type structure align with existing unicon.plugin for viptela.
If not, can you help merge this PR?

Reply:
@vanwang3 actually, we saw some issues with old way (os:sdwan, series:viptela) for corner case so we added `os: viptela` to unicon.plugin recently. so what I suggested was, should work and we would like to encourage to use new way.

https://pubhub.devnetcloud.com/media/unicon/docs/user_guide/supported_platforms.html#
![image](https://user-images.githubusercontent.com/34322875/81925243-40122300-95ae-11ea-9aa8-c0185d8e9837.png)

Can I ask you to confirm if below works in your env? This requires latest packages (20.4)
```
        vmanage:
            os: viptela
            custom:
                abstraction:
                    order: [os]
```
or viptela device has another device which we can access via REST like vManage? if yes, as I mentioned above, we may need to have `series: vmanage`.

Reply:
@tahigash Thanks! I didn't catch the unicon.plugin in 20.4 before. I have modified the organization of the directory/abstraction for my PR, can use as below now.
`        vmanage:
            os: viptela
            custom:
                abstraction:
                    order: [os]`

Reply:
@tahigash @jeaubin any more need to be modified?